src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+
+      {age && <p className="Person__age">I am {age}</p>}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {sex === 'm' ? `wife` : `husband`}
```

Comment:
```suggestion
          {partnerName} is my {sex === 'm' ? `wife` : `husband`}
```
let's create variable for this expression sex === 'm' ? `wife` : `husband`
`Also  <p className="Person__partner"> in both cases, only the content is different`
this comment didn't fixed, use only one `<p>` and change only content you don't need to repeat `<p>` two times